praw/models/reddit/redditor.py

```diff
@@ -301,6 +325,42 @@ def trophies(self) -> List["praw.models.Trophy"]:
         """
         return list(self._reddit.get(API_PATH["trophies"].format(user=self)))
 
+    def trust(self):
+        """Add the Redditor to your whitelist of trusted users.
+
+        Trusted users will always be able to send you PMs.
+
+        Example usage:
+
+        .. code-block:: python
+
+            reddit.redditor("AaronSw").trust()
+
+        Use the ``accept_pms`` parameter of :meth:`.Preferences.update` to toggle your
+        ``accept_pms`` setting between ``everyone`` and ``whitelisted``. For example:
+
+        .. code-block:: python
+
+            # Accept private messages from everyone:
+            reddit.user.preferences.update(accept_pms="everyone")
+            # Only accept private messages from trusted users:
+            reddit.user.preferences.update(accept_pms="whitelisted")
+
+        You may trust a user even if your ``accept_pms`` setting is switched to
+        ``everyone``.
+
+        .. note::
+
+            You are allowed to have a user on your blocked list and your friends list at
+            the same time. However, you cannot trust a user who is on your blocked list.
+
+        .. seealso::
+
+            :meth:`~.distrust`, :meth:`.Preferences.update`, and :meth:`.User.trusted`.
```

Comment:
```suggestion
            - :meth:`~.distrust`
            - :meth:`.Preferences.update`
            - :meth:`.User.trusted`
```
Can we make this a list?

Reply:
I originally wrote it the way you've suggested, but docstrfmt reformatted it so that all three were on the same line. I checked again, and docstrfmt doesn't complain if I format those items as a list with empty lines between the list entries.

Reply:
It shouldn't have an issue with the bullet list. Are you running pre_push.py?

Reply:
Yes, and you're right. I had forgotten to bullet the list.